gradlew

```diff
@@ -6,12 +6,30 @@
 ##
 ##############################################################################
 
-# Add default JVM options here. You can also use JAVA_OPTS and GRADLE_OPTS to pass JVM options to this script.
-DEFAULT_JVM_OPTS=""
+# Attempt to set APP_HOME
```

Comment:
Without a gradle verify job I would never trust this. e.g https://github.com/hannesa2/LiveEdgeDetection/blob/master/.github/workflows/gradle-wrapper-validation.yml

Reply:
This has come from the upstream version of Gradle. What threat model do you see from keeping this as is?

Reply:
A blueprint as a link is too easy ?

Reply:
Your statement doesn't seem to answer the question of what problem you're trying to solve with that change. To me there doesn't seem to be need to make your suggested change because it's not a requirement to achieve the goal of this PR.